upstream-community-operators/sysdig/sysdig-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,101 @@
+    mediatype: image/svg+xml
+  install:
+    spec:
+      deployments:
+      - name: sysdig-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: sysdig-operator
+          strategy: {}
+          template:
+            metadata:
+              creationTimestamp: null
+              labels:
+                name: sysdig-operator
+            spec:
+              containers:
+              - env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: sysdig-operator
+                image: docker.io/sysdiglabs/sysdig-operator:1.4.0
+                imagePullPolicy: Always
+                name: sysdig-operator
+                ports:
+                - containerPort: 60000
+                  name: metrics
+                resources: {}
+              serviceAccountName: sysdig-operator
+    strategy: deployment
+  maturity: Basic Install
```

Comment:
This should be removed.

Reply:
The same that Falco operator :stuck_out_tongue:  

I'm not sure about the entry/entries that should be removed. Are you referring only to the maturity entry?

Reply:
Yes, `spec.maturity` isn't required. When it is used, `spec.maturity` can only be set to certain values such as `alpha` or `stable`. It's safe to remove the field to avoid confusion.

Reply:
Great, thank you!